com.google.Earth.appdata.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>com.google.Earth</id>
```

Comment:
You could add `<launchable>`.

Reply:
makes sense, although doesn't that get added automagically in the build process? I should check.